src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    /**
+     * 
+     * @return a printable string representation of the object
```

Comment:
For a method with only return statement, you can remove `@return` and write the description as `Returns ...`.